language/stdlib/modules/SlidingNonce.move

```diff
@@ -18,16 +18,20 @@ module SlidingNonce {
 
     /// Grants the `CreateSlidingNonce` privilege to the calling `account`.
     /// Aborts if the calling account does not have the association root role.
-    public fun grant_privileges(account: &signer) {
-        Roles::add_privilege_to_account_association_root_role(account, CreateSlidingNonce{});
-    }
+    // public fun grant_privileges(account: &signer) {
+    // }
 
     /// Calls try_record_nonce and aborts transaction if returned code is non-0
     public fun record_nonce_or_abort(account: &signer, seq_nonce: u64) acquires SlidingNonce {
         let code = try_record_nonce(account, seq_nonce);
         assert(code == 0, code);
     }
 
+
+    public fun has_create_sliding_nonce_privilege(lr_account: &signer): bool {
+        has_libra_root_role(lr_account)
```

Comment:
Nit: The style we use use elsewhere in the stdlib is important just the module + using the module prefix at every call site (i.e., `Roles::has_libra_root_role`). That makes it easier to see where functions are defined.